debian/patches/0060-add-simd-optimized-tonemapx-filter.patch

```diff
@@ -3896,6 +4311,23 @@ Index: FFmpeg/libavfilter/vf_tonemapx.c
 +    av_log(s, AV_LOG_DEBUG, "Requested output format: %s\n",
 +           s->format_str);
 +
++    if (s->trc == AVCOL_TRC_SMPTE2084) {
```

Comment:
`:t=smpte2084:format=yuv420p` this should also be rejected.

```
Stream #0:0(und): Video: wrapped_avframe, yuv420p(pc, bt2020nc/bt2020/smpte2084, progressive)
```

// ref to gpu filters

```diff
+    if (out_trc == AVCOL_TRC_SMPTE2084) {
+        int is_10_or_16b_out = s->out_desc->comp[0].depth == 10 ||
+                               s->out_desc->comp[0].depth == 16;
+        if (!(is_10_or_16b_out &&
+            out_pri == AVCOL_PRI_BT2020 &&
+            out_spc == AVCOL_SPC_BT2020_NCL)) {
+            av_log(ctx, AV_LOG_ERROR, "HDR passthrough requires BT.2020 "
+                   "colorspace and 10/16 bit output format depth.\n");
+            return AVERROR(EINVAL);
+        }
+    }
```